src/main/resources/application.yaml

```diff
@@ -373,12 +514,6 @@ darts:
       dets-to-arm-batch-processor:
         max-arm-manifest-items: 100
         manifest-file-prefix: DETS
```

Comment:
there are 3 tasks recently added to the LLD called `ArmRpoPoll`, `ArmRpoReplay` and `ArmMissingResponseReplay` that don't look like were included in this change. Could we add them?

Reply:
They were not included in this config as we do not yet have automated tasks made for them in code.
I would expect the config to be added once the automated task is added in code